Documentation/operations/upgrade.rst

```diff
@@ -513,6 +513,21 @@ Helm Options
   or "false" as values, you should remove the quotes. For example in helm command, instead of
   ``--set-string disableEndpointCRD="true"``, it should be replaced by ``--set disableEndpointCRD=true``.
 
+Cilium CLI
+~~~~~~~~~~
+
+Please upgrade Cilium CLI to `v0.15.0 <https://github.com/cilium/cilium-cli/releases/tag/v0.15.0>`_
```

Comment:
```suggestion
Upgrade Cilium CLI to `v0.15.0 <https://github.com/cilium/cilium-cli/releases/tag/v0.15.0>`_
```
Can be avoid use of please? It doesn't always translate well to other languages.